tests/gpflow/covariances/test_multioutput.py

```diff
@@ -73,6 +78,12 @@ def test_kuf(inducing_variable, kernel):
     Kuf = mo_kufs.Kuf(inducing_variable, kernel, Datum.Xnew)
 
 
+@pytest.mark.parametrize("inducing_variable", multioutput_fallback_inducing_variable_list)
+def test_kuf_fallback_shared_inducing_variables(inducing_variable):
+    kernel = mk.LinearCoregionalization(make_kernels(Datum.L), Datum.W)
+    Kuf = mo_kufs.Kuf(inducing_variable, kernel, Datum.Xnew)
```

Comment:
I appreciate the other tests don't do this either, but... should we at least check the returned shape here? or clarify that it's just a smoke test "well it hasn't thrown an error :shrug:"?

Reply:
Good point - I have added shape asserts.